packages/visual-editor/src/utils/index.ts

```diff
@@ -41,3 +41,4 @@ export {
 } from "./migrate.ts";
 export { withPropOverrides } from "./withPropOverrides.ts";
 export { getLocationPath } from "./getLocationPath.ts";
+export { filterComponentsFromConfig } from "./filterComponents.tsx";
```

Comment:
I don't believe we should be exporting this if it will never be used by anyone externally. Looks like we've now contained it all internally, right?

Reply:
Nvm